### PR TITLE
Attributed string concatenation

### PR DESCRIPTION
**Pull request by josefdolezal**
This PR adds small extension to `NSAttributedString` allowing you to concatenate attributed strings. The concatenation may be done using method on `NSAttributedString` or by new concatenation operator `++`.

I also added swift version shield (with version 4.0) to readme -- until now, the required version of swift was 2.2.

### CODE REVIEW

File: ACKategories/Classes/NSAttributedStringExtensions.swift [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import Foundation
+
+precedencegroup ConcatenationGroup {
+    associativity: left
+    higherThan: AssignmentPrecedence
+}
+
+infix operator ++: ConcatenationGroup
+
+public func ++(_ lhs: NSAttributedString, _ rhs: NSAttributedString) -> NSAttributedString {
```
Comment (olejnjak) [inline ACKategories/Classes/NSAttributedStringExtensions.swift]: I am just wondering why not use ordinal `+` operator?

Comment (josefdolezal) [reply ACKategories/Classes/NSAttributedStringExtensions.swift]: I knew someone will ask! 😄 To be honest, no special reason. It's just more transparent that it's not part of the STD/Foundation to me.

Comment (olejnjak) [reply ACKategories/Classes/NSAttributedStringExtensions.swift]: That's a question I'd personally prefer standard `+` as all the stuff like precedence is already solved and the API is consistent.

What @pteasima thinks?

Comment (janmisar) [reply ACKategories/Classes/NSAttributedStringExtensions.swift]: I agree, let's use just `+` operator.

**Pull request merged**